external/plex/dist/Plex/test_tm.py

```diff
@@ -1,7 +1,7 @@
 import sys
 sys.stderr = sys.stdout
 
-from TransitionMaps import TransitionMap
+from Transitions import TransitionMap
```

Comment:
So, I'm instinctively against modifying third-party code, because I assume merging those modifications with upstream would be a nightmare.  @riastradh-probcomp , do you expect that with the way you set this up, my assumption is just wrong?  Also, should I assume that we will merge upstream patches rarely enough that my fear is irrelevant even if it is somewhat justified?